src/layer/Layer.Interactive.leafdoc

```diff
@@ -8,6 +8,10 @@ Use the [event handling methods](#evented-method) to handle these events.
 @option interactive: Boolean = true
 If `false`, the layer will not emit mouse events and will act as a part of the underlying map.
 
+@option bubblingEvents: Boolean = true
```

Comment:
How about calling the option `bubblingMouseEvents`, to make it more explicit which events it applies to (even though the doc string already says this)?